### PR TITLE
Add adapt credentials endpoint

### PR DESCRIPTION
**Pull request by c0d1ngm0nk3y**
# Pull Request Template

## Prerequisites

* This relates to #167 and implements the adapt_credentials endpoint.

## Motivation

One step closer to implementing the issue #167 

## Approach

It forwards the additional endpoint adapt_credentials to the service broker.

## Pull Request status

* [x] Initial implementation
* [x] Unit tests

## Third-party code

n/a

## Work in Progress label

n/a

### CODE REVIEW

**Comment by c0d1ngm0nk3y** [comment]
@dzahariev as discussed with @loewenstein

**Comment by loewenstein** [comment]
FYI @fmui

File: test/osb_test/osb_test.go [hunk 1]
```diff
@@ -445,6 +446,24 @@ var _ = Describe("Service Manager OSB API", func() {
 		})
 	})
 
+	Describe("Post Binding Adapt Credentials", func() {
```
Comment (KirilKabakchiev) [inline test/osb_test/osb_test.go]: Would it make sense to add the other cases as well? Currently we have call to working broker and call to broken broker. We might add the cases for verifying query params are preserved as well as calls to missing broker and stopped broker.

Comment (c0d1ngm0nk3y) [reply test/osb_test/osb_test.go]: @KirilKabakchiev Could you check if the added tests are sufficient from your point of view?

Comment (KirilKabakchiev) [reply test/osb_test/osb_test.go]: looks good, thanks

**Pull request merged**